EmbeddedPkg/Library/GdbSerialDebugPortLib/GdbSerialDebugPortLib.c

```diff
@@ -17,7 +17,8 @@
 #include <Protocol/DebugPort.h>
 
 EFI_DEBUGPORT_PROTOCOL  *gDebugPort = NULL;
-UINT32                  gTimeOut    = 0;
+// UINTN                   gTimeOut = 0;  // MU_CHANGE TCBZ3616
+UINT32  gTimeOut = 0;                     // MU_CHANGE TCBZ3616

```

Comment:
Looks like this may already have made it upstream?

Reply:
Yes, it was independently fixed here https://github.com/tianocore/edk2/commit/c05a218a9758225ddf94eedb365633f2154551da. Will drop it.